lib/config/options/index.ts

```diff
@@ -1998,6 +1998,7 @@ const options: RenovateOptions[] = [
     default: [],
     allowedValues: [
       'bundlerConservative',
+      'composerGitlabToken',
```

Comment:
@rarkins I'm not sure if this is the right way to solve it. 🤔


```suggestion
      'noComposerGitlabToken',
```
if we go this way, the value should reflect the behavior.